.github/latest_lts

```diff
@@ -0,0 +1 @@
+LTS
```

Comment:
Should we _also_ remove https://github.com/hazelcast/hazelcast-docker/blob/5.5.1/.github/latest_lts?

Reply:
Don't need it to as it won't be rebuilt after 5.5.2 release (we rebuild only the latest patch versions). There's a slight risk window but I've reworked this logic in the upcoming PR